### PR TITLE
Create Italian translation

### PR DESCRIPTION
**Pull request by albanobattistella**

### CODE REVIEW

File: po/it.po [hunk 1]
```diff
@@ -0,0 +1,377 @@
...
+"Last-Translator: Albano Battistella <albanobattistella@gmail.com>\n"
+"Language-Team: Italian <LL@li.org>\n"
+"Language: it\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+#: de.swsnr.turnon.desktop.in:3 resources/de.swsnr.turnon.metainfo.xml.in:5
+msgid "Turn On"
+msgstr "Turn On"
```
Comment (swsnr) [inline po/it.po:21]: Please translate the application name. Deepl suggests "Accendere"

File: po/it.po [hunk 2]
```diff
@@ -0,0 +1,377 @@
...
+msgid "Add new device"
+msgstr "Aggiungi nuovo dispositivo"
+
+#: resources/de.swsnr.turnon.metainfo.xml.in:8
+msgid ""
+"Conveniently turn on devices in your network, such as a NAS or a gaming "
+"console, using Wake On LAN (WoL)."
+msgstr ""
+"Accendi comodamente i dispositivi nella tua rete, come un NAS o una console " 
+"di gioco,utilizzando Wake On LAN (WoL)."
```
Comment (swsnr) [inline po/it.po:41]: I'm not familiar with Italian, but shouldn't there be a space after this comma?

**Review by swsnr — CHANGES_REQUESTED** [review]

**Comment by swsnr** [comment]
Please also update the `LINGUAS` file in this PR.

File: po/it.po (hunk 1 above)
Comment (albanobattistella) [reply po/it.po:21]: > Please translate the application name. Deepl suggests "Accendere"

Hi,
not good.
Turn on is ok.
thanks.
A.

File: po/it.po (hunk 2 above)
Comment (albanobattistella) [reply po/it.po]: HI,
fixed.
I cannot do a single pull request, I don't know why.

Comment (swsnr) [reply po/it.po]: Don't use the gitlab Web interface.

File: po/it.po (hunk 1 above)
Comment (swsnr) [reply po/it.po:21]: Why can't you translate the application name?

File: po/it.po (hunk 2 above)
Comment (swsnr) [reply po/it.po]: Or alternatively, use the Web Interface to make a commit to the patch-1 branch in your fork; that'll make it appear in this PR.

These are got basics, they shouldn't give you trouble when you intend to contribute on Github 😬

**Pull request closed without merging**